packages/expo-font/ios/EXFont/Singletons/EXFontScalersManager.m

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXFont/EXFontScalersManager.h>
+
+#import <EXCore/EXDefines.h>
+#import <EXFont/EXFont.h>
+
+#import <objc/runtime.h>
+
+static dispatch_once_t initializeCurrentFontScalersOnce;
```

Comment:
Since this shouldn't be accessed from methods aside from `+[initialize]`, it'd be better as a static variable inside the method.